codeHF/config_tasks.sh

```diff
@@ -415,9 +416,9 @@ EOF
 function MakeScriptPostprocess {
   POSTEXEC="echo Postprocessing"
   # Compare AliPhysics and O2 histograms.
-  [[ $DOALI -eq 1 && $DOO2 -eq 1 ]] && {
+  [[ $DOPOSTPROCESS -eq 1 ]] && {
```

Comment:
Why do you want to change this?

Reply:
This is for a newcomer to the code (like me yesterday) far from obvious. The name of this setter at the top of the script suggests that it also take care of enabling/disabling the plotting itself.

I would suggest to make it an `(DOALI and DOO2) or DOPOSTPROCESS` condition. What do you think?

Reply:
If `DOPOSTPROCESS=0`, then the function `MakeScriptPostprocess` is not executed at all, so adding an `or DOPOSTPROCESS` condition inside it is equivalent to removing the current condition.
The main documentation says: "This step typically compares AliPhysics and O2 output and produces plots." but feel free to suggest a better phrasing.
Concerning the comment "(Compare AliPhysics and O2 output.)" in the script, I'm fine with removing it for clarity.
Would that be fine?